### ISSUE TITLE
2FA: Disable autocomplete on the authentication code field

### ISSUE THREAD

**Issue by alexwlchan** [pypa/warehouse#5805]
**Describe the bug**
When I log in with 2FA enabled, my browser offers 2FA codes I’ve used in the past as autocomplete suggestions. Screenshots below from Safari and Chrome.

![Screenshot 2019-05-07 at 17 47 43](https://user-images.githubusercontent.com/301220/57317686-9a9db700-70f0-11e9-899d-521db8da84ac.png)

![Screenshot 2019-05-07 at 17 50 12](https://user-images.githubusercontent.com/301220/57317692-9c677a80-70f0-11e9-812a-a27cea1d4eae.png)

**Expected behavior**
Don’t do that.

**To Reproduce**

1. Enable 2FA
2. Log out of your account
3. Log back in, and pay attention to browser autocomplete on the 2FA screen

**My Platform**
I’m using macOS 10.14.2. I can reproduce this behaviour in Safari 12.0.2 and Chrome 74.0.3729.131.

**Additional context**
This is the `<input>` field provided by PyPI on that screen (line breaks added for readability):

```html
<input
 autocapitalize="off"
 autocorrect="off"
 autofocus=""
 class="form-group__input"
 id="totp_value"
 name="totp_value"
 placeholder="Authentication code"
 required="required"
 spellcheck="false"
 tabindex="1"
 type="text"
 value="">
```

Here's an equivalent field in FastMail, my email provider, which doesn’t offer autocomplete suggestions:

```html
<input
 id="v24-input"
 class="v-Text-input"
 type="text"
 name="totp"
 autocomplete="off"
 pattern="[0-9]*"
 maxlength="6"
 placeholder="XXXXXX">
```

The `autocomplete="off"` looks like the bit to steal here – although the other attributes might not be unhelpful either!

**Comment by dstufft** [comment]
We should totally do this and a PR that adds it would be much appreciated!

**Comment by di** [comment]
Fixed in #5807, thanks @alexwlchan!
